README.md

```diff
@@ -90,6 +91,7 @@ gnag {
   - ***enabled*** - set if the android lint reporter should look for a lint report
   - ***severity*** - can be 'Error' or 'Warning' depending on which severity you want Gnag to check
 - ***github*** - block to customize GitHub reporting (only used during the `gnagReport` task
+  - ***rootUrl*** - root URL to use when communicating with the GitHub api, if not provided will default to "https://api.github.com/repos/"
```

Comment:
Might want to explicitly call out that the `rootUrl` needs to include that trailing slash.